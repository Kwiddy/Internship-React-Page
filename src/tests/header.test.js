import renderer from "react-test-renderer";
import React from "react";
import Header from "../components/header.js";

describe("Header component", () => {
  it("renders correctly", () => {
    const headText = { mainText: "Your account", subText: "Linked services" };
    const rendered = renderer.create(<Header headText={headText} />);
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
