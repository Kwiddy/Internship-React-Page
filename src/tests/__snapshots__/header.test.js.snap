// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<header>
  <div
    className="headText"
  >
    <h2
      className="MuiTypography-root MuiTypography-h1"
      fontSize="1"
      id="mainHead"
    />
    <h2
      className="MuiTypography-root MuiTypography-h2 MuiTypography-gutterBottom"
      id="subHead"
    />
  </div>
</header>
`;
